src/ServiceControl.Config/Validation/ValidationTemplate.cs

```diff
@@ -99,7 +100,12 @@ void Validate(object sender, PropertyChangedEventArgs e)
             if (properties.Contains(e.PropertyName))
             {
                 validationResults.RemoveAll(x => x.PropertyName == e.PropertyName);
-                var validationResult = validator.Validate(target, e.PropertyName);
+                var validationContext = new ValidationContext<RxPropertyChanged>(
```

Comment:
Seem to be correct according to their comment [here](https://github.com/FluentValidation/FluentValidation/issues/183#issuecomment-168676295).